app/scrapers/__init__.py

```diff
@@ -25,7 +25,6 @@ def small_test():
 
 def feedgen(query, engine, count = 10):
     urls = scrapers[engine].search(query, count)
-    result = urls
-    print(result)
-    print(len(result))
-    return result
+    print(urls)
```

Comment:
We should remove these print statements as well what do you say ?

Reply:
I do not have an opinion one way or the other on printing.

Reply:
@cclauss i agree with @AnshulMalik